ls1intum/Artemis
# `General`: Add new contributors to about us page

### Checklist
#### General

### Motivation and Context
Adds new contributors to the about us page.

### How to add yourself
1. Checkout the `chore/about-us` branch locally
2. Add yourself to this file: `src/main/resources/public/content/about-us.json`
 - Use `fullName` for your name
 - Use `sortBy` if the automatic sorting does not detect your last name correctly (if your name consists of mulitple words)
3. Add your image to the `public/images/about` directory and link that image using the `photoDirectory` key. Make sure to use a square, otherwise crop your image before uploading it.

### Steps for Testing
View the about us page and check that the new contributors are visible and sorted.
